code/media/src/phone/java/com/adobe/marketing/mobile/MediaConstants.java

```diff
@@ -37,57 +37,57 @@ private StreamType() {}
+        public static final String ARTIST = "artist";
+        public static final String AUTHOR = "author";
+        public static final String LABEL = "label";
+        public static final String PUBLISHER = "publisher";
+        public static final String STATION = "station";
 
         private AudioMetadataKeys() {}
     }
 
     /** These constant strings define standard video metadata keys. */
     public static final class VideoMetadataKeys {
-        public static final String SHOW = "a.media.show";
-        public static final String SEASON = "a.media.season";
-        public static final String EPISODE = "a.media.episode";
-        public static final String ASSET_ID = "a.media.asset";
-        public static final String GENRE = "a.media.genre";
-        public static final String FIRST_AIR_DATE = "a.media.airDate";
-        public static final String FIRST_DIGITAL_DATE = "a.media.digitalDate";
-        public static final String RATING = "a.media.rating";
-        public static final String ORIGINATOR = "a.media.originator";
-        public static final String NETWORK = "a.media.network";
-        public static final String SHOW_TYPE = "a.media.type";
-        public static final String AD_LOAD = "a.media.adLoad";
-        public static final String MVPD = "a.media.pass.mvpd";
-        public static final String AUTHORIZED = "a.media.pass.auth";
-        public static final String DAY_PART = "a.media.dayPart";
-        public static final String FEED = "a.media.feed";
-        public static final String STREAM_FORMAT = "a.media.format";
+        public static final String AD_LOAD = "adLoad";
+        public static final String ASSET_ID = "assetID";
+        public static final String AUTHORIZED = "isAuthenticated";
+        public static final String DAY_PART = "dayPart";
+        public static final String EPISODE = "episode";
+        public static final String FEED = "feed";
+        public static final String FIRST_AIR_DATE = "firstAirDate";
+        public static final String FIRST_DIGITAL_DATE = "firstDigitalDate";
+        public static final String GENRE = "genre";
+        public static final String MVPD = "mvpd";
+        public static final String NETWORK = "network";
+        public static final String ORIGINATOR = "originator";
+        public static final String SEASON = "season";
+        public static final String SHOW = "show";
+        public static final String SHOW_TYPE = "showType";
+        public static final String STREAM_FORMAT = "format";
+        public static final String RATING = "rating";
 
         private VideoMetadataKeys() {}
     }
     /** These constant strings define standard player states. */
     public static final class PlayerState {
-        public static final String FULLSCREEN = "fullscreen";
-        public static final String PICTURE_IN_PICTURE = "pictureInPicture";
         public static final String CLOSED_CAPTION = "closedCaptioning";
```

Comment:
closeCaption

Reply:
Tagging this document for reference: https://experienceleague.adobe.com/docs/media-analytics/using/implementation/variables/player-state-parameters.html?lang=en

Reply:
I can update this to be the same as the iOS implementation, but this should be "closedCaptions" or "closedCaptioning".   The "closed" means the captions are not visible to the screen/user.